sdk/core/azure-core/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+- Added option `BUILD_CRYPTO` to support building azure-core without the Cryptographic namespace (no hashing). ON by default.
```

Comment:
Can you help me understand why we want to expose this build option to the customer? What scenario requires disabling building this namespace and why?

Reply:
> Crypto / Hashing : MD5 and Sha hashing is currently exposed on the public surface and implemented by calling OpenSSL lib. This PR is making all the Cryptographic namespace an optional feature that can be opt-out when building the azure-core lib

Oh... it's because of hashing :(
We got rid of it from base64 and uuid, but we still have one component left